lib/tasks/maintenance.rake

```diff
@@ -22,4 +22,10 @@ namespace :maintenance do
       end
     end
   end
+
+  task :clear_discarded_api_actions => [:environment] do 
```

Comment:
after fixing the tenant access issue: 
see more about accessing all tenants in Rake tasks here: https://github.com/restarone/violet_rails/blob/master/lib/tasks/report.rake

@Pralish this needs to be added to the `schedule.rb` run it once a day